lib/notion_to_md/blocks/block.rb

```diff
@@ -32,8 +32,12 @@ def initialize(block:, children: [])
       #
       def to_md(tab_width: 0)
         block_type = block.type.to_sym
-        md = Types.send(block_type, block[block_type])
-        md + build_nested_blocks(tab_width + 1)
+        if block_type == :toggle
```

Comment:
Nice solution! I was looking into this method before. So the idea is that if it is a toggle, we get the content and manually build the nested blocks?

Reply:
Exactly 👍